capstone/src/main/java/com/google/univiz/collegescorecarddata/SearchData.java

```diff
@@ -0,0 +1,8 @@
+package com.google.univiz.collegescorecarddata;
+
+import java.util.List;
+
+public interface CollegeScoreCard {
```

Comment:
This interface name is wrong (and actually won't build).

Reply:
yep-sorry for that oversight!